app/src/main/java/com/beemdevelopment/aegis/importers/GoogleAuthImporter.java

```diff
@@ -114,12 +123,22 @@ public Entry(Cursor cursor) {
             _email = SqlImporterHelper.getString(cursor, "email", "");
             _issuer = SqlImporterHelper.getString(cursor, "issuer", "");
             _counter = SqlImporterHelper.getLong(cursor, "counter");
+            if(cursor.getColumnIndex("isencrypted") != -1 && SqlImporterHelper.getInt(cursor, "isencrypted") > 0) {
+                _isencrypted = true;
+            } else {
+                _isencrypted = false;
+            }
```

Comment:
An inline expression is a bit shorter. Also ``_isEncrypted`` instead of ``_isencrypted``.

```java
_isEncrypted = cursor.getColumnIndex("isencrypted") != -1 && SqlImporterHelper.getInt(cursor, "isencrypted") > 0
```